public/index.html

```diff
@@ -21,42 +21,33 @@
+            <div class="col text-center" style="padding-right:20px; padding-left:10px; background: radial-gradient(circle, #bdd5bd, #e4ece4); border-radius: 15px;">
+                <h4> Create a navPlace object </h4>
                 <p>
-                    The IIIF Presentation API is extendable and benefits from its active community through <a href="https://iiif.io/api/extension/" target="_blank">community driven extensions</a>.  The <a target="_blank" href="https://iiif.io/community/groups/maps/">IIIF Maps</a> groups developed a foundation around <a target="_blank" href="https://geojson.org/geojson-ld/">GeoJSON-LD</a> and designed ways to use this data format within the IIIF APIs and data schemas.  The <a target="_blank" href="https://iiif.io/api/extension/navplace/">navPlace Extension</a> formally introduced a <code>navPlace</code> property which is designed to house a single GeoJSON Feature Collection thus making a formal and direct connection between the resource and place.  This path will result in a formatted <code>navPlace</code> JSON object which can be used with your resource.
-                </p>
+                    The navPlace property is for web resources that are IIIF Presentation API Defined Types. <a target="_blank" href="https://iiif.io/api/presentation/3.0/#21-defined-types".> </a> Learn more about IIIF and the navPlace property by exploring <a target="_blank" href="https://iiif.io/api/extension/navplace/">navPlace Extension</a>.
+                <br></br>
                 <a class="button primary" style="display: inline-block;" href="navPlace.html">
                     Create a navPlace Object
                 </a>
-                <h5> Annotation </h5>
-                <p>
-                    The IIIF Presentation API incorporates the <a target="_blank" href="https://www.w3.org/TR/annotation-model/">W3C Web Annotation</a> for the "Annotation" objects created under its schema.  These Annotations can be embedded directly onto <a target="_blank" href="https://iiif.io/api/presentation/3.0/#21-defined-types">IIIF Defined Types</a>.  It can also target those data types as non-destructive, non-dispruptive independent data assertion.  This path seeks to finish with a real active <abbr title="Linked Open Usable Data">LOUD</abbr> Geolocating Web Annotation for the resource you provide.
-                </p>
-                <a class="button primary" style="display: inline-block;" href="annotate.html">
-                    Create a Geolocating Web Annotation
-                </a>
             </div>
-        </div>
-        <div class="row">
-            <div class="col">
-                <h4> Other Linked Open Data Resources </h4>
+        
+        
+            <div class="col text-center" style="padding-left:20px; padding-right:10px; background: radial-gradient(circle, #bdd5bd, #e4ece4); border-radius: 15px;">
+                <h4> Create a Geolocating Web Annotation </h4>
+                
                 <p>
```

Comment:
This <p> doesn't close! When you close it, you can give it a padding or margin to get the spacing effect you are going for (this is in relation to the br comment).